src/nav/apm.yml

```diff
@@ -7,7 +7,7 @@ pages:
     pages:
     - title: APM best practice guide
       path: /docs/new-relic-solutions/best-practices-guides/full-stack-observability/apm-best-practices-guide
-    - title: Triage with APM
+    - title: Begin with APM
```

Comment:
Few thoughts. Maybe something like: 

Start troubleshooting with APM (and maybe to realign with actual doc title, change title to 'Start troubleshooting with the APM summary page)
Maybe we move this doc up above best practice guide, because I think most customers would likely want to learn some basics of using APM and then move on to best practices

Reply:
Done